@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -920,6 +909,25 @@ watch(
   },
   { immediate: true }
 )
+
+// BOOT FIRMWARE SELECTOR
+
+const bootFirmwareOptions = ['bios', 'uefi']
+
+const { id: bootFirmwareSelectId } = useFormSelect(bootFirmwareOptions, {
+  model: toRef(vmState, 'bootFirmware'),
+})
+
+watch(
+  () => vmState.bootFirmware,
```

Comment:
Is the purpose of this `watch` to set the default value of `copyHostBiosStrings` when the user selects a template? If so, shouldn't it be `watch`ing `new_vm_template` instead of `bootFirmware`?